README.md

```diff
@@ -19,12 +19,11 @@ provided under the `dependencies` directory for each platform (if applicable).
 __Note__: CentOS and Ubuntu use the version of OpenSSL and zlib provided with the
 distribution:
 
-* [CentOS 6][centos-6-dependencies]
 * [CentOS 7][centos-7-dependencies]
-* [CentOS 8][centos-8-dependencies]
-* [Ubuntu 14.04][ubuntu-14-04-dependencies]
-* [Ubuntu 16.04][ubuntu-16-04-dependencies]
-* [Ubuntu 18.04][ubuntu-18-04-dependencies]
+* [Rocky Linux 8][rocky-linux-8-dependencies]
+* [Rocky Linux 9][rocky-linux-9-dependencies]
```

Comment:
Might want to consider whether we want major.minor for Rocky releases or just major here.